std_msgs/Header header
int32 xian_dj_tele_op_params_server_heart_beat 
int32 xian_dj_tele_op_controller_server_heart_beat 
int32 xian_dj_tele_op_controller_server_tcp_heart_beat 
int32 xian_dj_tele_op_controller_client_heart_beat 
int32 xian_dj_tele_op_controller_client_tcp_heart_beat 
int32 xian_dj_tele_op_error_monitor_heart_beat 
int32 xian_dj_tele_op_back_end_heart_beat 
int32 xian_dj_tele_op_left_client_cmd 
int32 xian_dj_tele_op_right_client_cmd 
int32 xian_dj_tele_op_up_client_cmd 
int32 xian_dj_tele_op_down_client_cmd
int32 xian_dj_tele_op_x_client_cmd 
int32 xian_dj_tele_op_b_client_cmd 
int32 xian_dj_tele_op_y_client_cmd 
int32 xian_dj_tele_op_a_client_cmd 
float32 xian_dj_tele_op_left_rocker_x_client_cmd 
float32 xian_dj_tele_op_left_rocker_y_client_cmd 
float32 xian_dj_tele_op_right_rocker_x_client_cmd 
float32 xian_dj_tele_op_right_rocker_y_client_cmd 
int32 xian_dj_tele_op_r1_client_cmd 
int32 xian_dj_tele_op_r2_client_cmd 
int32 xian_dj_tele_op_l1_client_cmd 
int32 xian_dj_tele_op_l2_client_cmd 
int32 xian_dj_tele_op_left_server_cmd 
int32 xian_dj_tele_op_right_server_cmd 
int32 xian_dj_tele_op_up_server_cmd 
int32 xian_dj_tele_op_down_server_cmd
int32 xian_dj_tele_op_x_server_cmd 
int32 xian_dj_tele_op_b_server_cmd 
int32 xian_dj_tele_op_y_server_cmd 
int32 xian_dj_tele_op_a_server_cmd 
float32 xian_dj_tele_op_left_rocker_x_server_cmd 
float32 xian_dj_tele_op_left_rocker_y_server_cmd 
float32 xian_dj_tele_op_right_rocker_x_server_cmd 
float32 xian_dj_tele_op_right_rocker_y_server_cmd 
int32 xian_dj_tele_op_r1_server_cmd 
int32 xian_dj_tele_op_r2_server_cmd 
int32 xian_dj_tele_op_l1_server_cmd 
int32 xian_dj_tele_op_l2_server_cmd 
int32 xian_dj_tele_op_controller_server_error 
int32 xian_dj_tele_op_controller_client_error 
int32 xian_dj_tele_op_controller_server_tcp_error 
int32 xian_dj_tele_op_controller_client_tcp_error 
int32 xian_dj_tele_op_error_code